python/iog/requirements.txt

```diff
@@ -17,7 +17,7 @@ networkx==2.6.2
 numpy
 opencv-python==4.5.2.52
 pathspec==0.8.1
-Pillow==8.2.0
+Pillow==9.0.0
```

Comment:
Have you tested whether IOG still works after the upgrade and a fresh package install?

Reply:
Yes I did

Reply:
OK perfect ❤️